CONTRIBUTING.md

```diff
@@ -188,6 +188,24 @@ Follow the example for any of the other pathway reconstruction algorithm.
 First pull the image `<username>/local-neighborhood` from Docker Hub.
 Then build the Docker image using the `Dockerfile` that was completed in Step 2.
 
+Modify Generate Inputs:
```

Comment:
Lets use the "``" syntax for filenames in the new instructions below. Please also add the names of the text files to modify.